mark-me-up-1.html

```diff
@@ -1,11 +1,24 @@
-This is basic HTML file. But right now, all it has is some text dumped into it.
-Your mission is to mark up this text so that it is more structured, readable and
-semantic. 
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8" />
+        <title>Mark Me Up</title>
+    </head>
+    <body>
+        <p id="top-paragraph">
+            This is basic <abbr>HTML</abbr> (HyperText Markup Language) file. But right now, all it has is some text dumped into it. Your mission is to mark up this text so that it is more structured, readable and semantic.
```

Comment:
A `title` attribute on the `abbr` gives extra hint as tooltip for visual users and also the fullform to screen readers